agreements/project-contribution/asset-transfer-agreement.md

```diff
@@ -0,0 +1,216 @@
+- `__________________________________________________________`  
+
+*Principal Office or Primary Address of Assignor(s):*
+
+`____________________________________________________________`  
+`____________________________________________________________`  
+`____________________________________________________________`  
+`____________________________________________________________`  
+
+*Assignor Email:* `________________________________________`
+
+---
+
+**Sponsor:** *Commonhaus Foundation, Inc.*
+
+*Sponsor Address:*  
+6526 OLD BRICK ROAD, #120-175  
+WINDERMERE, FL 34786  
+
+*Sponsor Email:* `legal@commonhaus.org`
+
+---
+
+This Asset Transfer Agreement ("Transfer Agreement") is made by and between Commonhaus Foundation ("Sponsor") and the Assignor(s) (“Assignor”) on behalf of the Project.
+
+## Recitals
+
+This Agreement governs the transfer of assets from the Assignor to the Sponsor in furtherance of the purposes and goals described in the [Fiscal Sponsorship Agreement][fs-landing].
+
+The intangible assets associated with the Project, as described below, will be transferred to the Sponsor, and the Sponsor will assume responsibility for the Project’s operations.
+After the transfer, the Assignor will no longer have any operational responsibility for the Project, except where other agreements apply (e.g., if the Assignor is also a Signatory of the Fiscal Sponsorship Agreement or continues to provide services to those Signatories).
+
+Operational responsibilities, including the ongoing management and quality standards for these assets, are further outlined in the [Fiscal Sponsorship Agreement][fs-landing] and [Terms and Conditions][].
+
+## Definitions
+
+All capitalized terms used in this Transfer Agreement not defined in it are as defined in the [Fiscal Sponsorship Agreement][fs-landing].
+
+**Assets**: The intangible assets needed by Project for it to operate under the auspices of Sponsor without any disruption, alteration, or modification of the operations, processes, or activities.
+The attached [schedules of assets](#schedules) are intended to provide a comprehensive list of all assets being transferred; however, the schedules are not binding and the parties retain the right to amend the schedules as necessary.
+
+**Contributor License Agreements**: The written agreements pursuant to which the Assignor has received licenses from others that allow for the reproduction, modification or distribution of third party contributions made to the Software.
+
+**Copyrights**: The copyrights owned by Assignor for the Software, including registrations, applications, renewals, and all related past and future infringement claims.
+
+**Domain Names**: The internet domain names used by the Project.
+
+**Hosting Accounts**: The accounts used by the Project for hosting of its software and related operations, such as web hosting accounts, version control repositories, document hosting platforms, build and testing servers, and cloud infrastructure.
```

Comment:
Does this include Discord servers?

Reply:
The Discord server would definitely be included in an inventory of assets associated with the project. If that Discord server is the official Discord Server for the project, it probably should. 

That does not mean the foundation would take control of that server, however. Definitely something to talk about as we go through terms and boundaries, etc.